frontend/packages/console-app/src/components/pdb/PDBForm.tsx

```diff
@@ -48,10 +50,24 @@ const getSelectedRequirement = (requirement: string, items: RequirementItems): s
   return i18next.t('console-app~Requirement');
 };
 
+function checkAvailabilityRequirementValue(formValues: FormValues, replicasCount: number) {
+  const { maxUnavailable, minAvailable } = formValues;
+  const minAvailableToStr = String(minAvailable);
+
+  if (minAvailableToStr.includes('%')) {
+    return parseInt(minAvailableToStr.slice(0, -1), 10) >= 100;
```

Comment:
seems like `parseInt` ignores trailing non-numerical characters so it should be safe to omit the string slicing

also this might not work for some locales if they [typed the percent sign first](https://en.wikipedia.org/wiki/Percent_sign#Form_and_spacing)

Reply:
> seems like `parseInt` ignores trailing non-numerical characters so it should be safe to omit the string slicing
> 
> also this might not work for some locales if they [typed the percent sign first](https://en.wikipedia.org/wiki/Percent_sign#Form_and_spacing)

@logonoff This change has nothing to do with actual values sent to the API server or locales. It is  just for user experience purposes.  Like other console forms, the users can send any value, and the server would accept or send an error.

Reply:
> It's still nice to have the frontend validation match the backend one closer

This is not a validation per se. Just an info to guide users. I rest my case.